hooks/useRailgunProvider.tsx

```diff
@@ -0,0 +1,28 @@
+// import { loadProviders } from "@/utils/railgun";
```

Comment:
This should probably be removed

Reply:
thanks! 26011e3

Reply:
`loadProviders` is now used, I'd say we can resolve this